grove/qft/tests/test_inverse_qft.py

```diff
@@ -0,0 +1,50 @@
+##############################################################################
+#
+#    A collection of tests for the inverse quantum Fourier transform.
```

Comment:
you can count this as a start of a `test_qft.py` so that we can add more tests to it rather than needing its own module for testing the inverse qft function